internal/cli/cli.go

```diff
@@ -78,8 +76,8 @@ func (c *Cli) Execute(controller controller.ControllerInterface) {
 func createInitCommand(controller controller.ControllerInterface) *cobra.Command {
 	initCmd := &cobra.Command{
 		Use:   "init",
-		Short: "Login to an existing KintoHub account",
-		Long:  `Helps you to log in an already existing KintoHub account`,
+		Short: "Init the cli",
+		Long:  `Create a 'kinto.yaml' file in your home directory`,
```

Comment:
`init` does not create a kinto.yaml file. as of now it is just for setting/resetting the master host